src/SFA.DAS.AssessorService.Web/Views/Certificate/Option.cshtml

```diff
@@ -47,10 +47,8 @@ else
                 }
 
                 <h1 class="govuk-heading-xl">
-                    @Localizer["Heading"]
+                    Choose the option for @Model.Standard
```

Comment:
You can keep the Localizer in like here: https://github.com/SkillsFundingAgency/das-assessor-service/blob/SV-456-StandardVersionRecordOutcome/src/SFA.DAS.AssessorService.Web/Views/Certificate/Version.cshtml#L50